### PR TITLE
misc. grammar, usage, style, and factual updates

### PR DESCRIPTION
**Pull request by horwitz**

### CODE REVIEW

File: src/data.json [hunk 1]
```diff
@@ -219,15 +219,15 @@
         "atomic_number": "28",
         "media": "media/28nickel.gif",
         "name": "Nickel",
-        "phrase": "**Ni**ckel. In 1928, the year before the Great Depression, people had more money (a nickel) than in the next year ('29). The U.S. coin called a \"nickel\" used to be made of nickel.",
+        "phrase": "**Ni**ckel. In 1928, the year before the Great Depression, people had more money (a nickel) than in the next year ('29). The U.S. coin called a \"nickel\" is (partly) made of nickel.",
```
Comment (horwitz) [inline src/data.json:222]: Even as early at the 1866 Shield nickel—the first US 5-cent coin generally referred to as a "nickel"—the coin was 75% Cu and 25% Ni (as it is today).

(Technically, the coin has changed composition along the way—e.g., during WWII—but that needn't affect the memory-peg phrase.)

File: src/data.json [hunk 2]
```diff
@@ -691,7 +691,7 @@
         "atomic_number": "87",
         "media": "media/87francium.gif",
         "name": "Francium",
-        "phrase": "**Fr**ance's Eiffel Tower. Built for the World's Fair and commemorated the centenniel of the French Revolution which technically began in 17**87**.",
+        "phrase": "**Fr**ance's Eiffel Tower. Built for the World's Fair and commemorated the centennial of the French Revolution, which some might say began in 17**87**.",
```
Comment (horwitz) [inline src/data.json:694]: I can't find any argument that the French Revolution began in 1787; I'm not particularly excited by the weasely text I use in the memory peg, but at least it isn't factually incorrect. (I didn't want to make a major change—e.g., use a completely different idea for the peg—at least not in this PR.)

File: src/data.json [hunk 3]
```diff
@@ -779,15 +779,15 @@
...
+        "phrase": "**C**auli-**f**lower. Remember Berkeley, California, are consecutive and that there is a 98 hidden in the cauliflower. California produces a lot of cauliflower.",
         "symbol": "Cf",
         "guid": "ILOhA]ha@}"
     },
     {
         "atomic_number": "99",
         "media": "media/99einsteinium.gif",
         "name": "Einsteinium",
-        "phrase": "Einstein's **es**oteric earrings. The hidden meaning is that his theory predicts you can only go 99% of the speed of light.",
+        "phrase": "Einstein's **es**oteric earrings. The hidden meaning is that his theory predicts you can only go ~99% of the speed of light.",
```
Comment (horwitz) [inline src/data.json:790]: Another update (like Fr) to make the text technically correct...

File: src/data.json [hunk 4]
```diff
@@ -171,7 +171,7 @@
         "atomic_number": "22",
         "media": "media/22titanium.gif",
         "name": "Titanium",
-        "phrase": "**Ti**tanic. The Titanic had 2200 passengers when it hit an iceberg and sank.",
+        "phrase": "**Ti**tanic. The Titanic had ~2200 passengers when it hit an iceberg and sank.",
```
Comment (horwitz) [inline src/data.json:174]: Another update (like Fr) to make the text technically correct...

**Comment by remiberthoz** [comment]
Thank you a lot, awesome PR !

I never really took the time to fact-check all these claims... It's great someone did. I'm a bit resilient to change the sentences, because once one starts learning it's confusing. But I agree that the facts should be correct. Plus, your suggestions are not changing anything to the learning process !

I'll merge and then publish a new version to Anki.

File: src/data.json (hunk 2 above)
Comment (remiberthoz) [reply src/data.json:694]: I will modify this one, stating that the construction of the Eiffel tower started in 1**8**87 with no ambiguity!

**Pull request merged**